### PR TITLE
Setup branch

### PR DESCRIPTION
**Pull request by ilyassatlassi**
In this activity i will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,138 @@
-# leaderboard
\ No newline at end of file
+# To Do List 📝
+
```
Comment (rotimiazeez) [inline README.md]: - Kindly add the correct project name to the header of your README file. 👍🏽

File: README.md [hunk 2]
```diff
@@ -1 +1,138 @@
...
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="120"  height="auto"/>
+  <br/>
+
```
Comment (rotimiazeez) [inline README.md:7]: - Please note the logo on your README file is broken. I'd suggest that you ME file. 😉

File: README.md [hunk 3]
```diff
@@ -1 +1,138 @@
...
+- HTML.
+- CSS.
+- JS.
+- Webpack
+
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+[To Do List 📝](https://ilyassatlassi.github.io/webpack/)
+
```
Comment (rotimiazeez) [inline README.md]: - Kindly remove the current live demo link, as it's for another project and add the current one or add coming soon.. if you don't have one yet. 👍🏽

File: README.md [hunk 4]
```diff
@@ -1 +1,138 @@
...
+- Kindly check the branches names in github then add them manually on your local machine, after that you may just pull the changes.
+
+### How to access the database?
+
+- It's not available for now I'm still working on it, once it's available I'll update this answer.
+
+# 📝 License
+
+This project is MIT licensed.
```
Comment (rotimiazeez) [inline README.md]: - Kindly provide a valid MIT licensed link for this project 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @ilyassatlassi  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️
- Good project setup 👏🏽
- Good UI 👍🏽 
- Good commit message(s) 😉
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:
### Adding a more descriptive PR description by adding a title and summary of what was done in the PR. This makes your PR look more professional 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (ilyassatlassi) [reply README.md:7]: @rotimiazeez Please what you mean about ME file

**Review by DuaneDave — COMMENTED** [review]
Hi @ilyassatlassi 👋,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ PR summary present
✔️ Readme is great
✔️ Linters are passing

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**